src/main/java/Duke.java

```diff
@@ -6,5 +8,99 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println("\t____________________________________________________________");
+        taskTracker();
+        System.out.println("\tBye. Hope to see you again soon!");
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public static void taskTracker() {
+        Task[] tasks = new Task[100];
```

Comment:
Hi, it's great that an object array is created.
However, I believe that magic numbers should be replaced with an appropriate name to better identify them according to the coding standards.